docs/running-on-kubernetes.md

```diff
@@ -477,6 +477,22 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
+  <td><code>spark.kubernetes.executor.labels</code></td>
+  <td>(none)</td>
+  <td>
+    Custom labels that will be added to the executor pods. This should be a comma-separated list of label key-value pairs,
+    where each label is in the format <code>key=value</code>.
```

Comment:
What I propose above would be a breaking change so that we don't have to support both at once. But we can also deduplicate fairly easily and throw an exception.